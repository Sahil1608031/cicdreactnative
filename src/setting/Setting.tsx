import React, { JSX } from 'react';
import { View, Text, StyleSheet } from 'react-native';

const Setting = (): JSX.Element => {
  return (
    <View style={styles.container}>
      <Text>Setting</Text>
    </View>
  );
};
const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
});
export default Setting;
