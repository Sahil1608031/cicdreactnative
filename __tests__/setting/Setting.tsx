import React from "react";
import { render } from "@testing-library/react-native";
import Setting from "../../src/setting/Setting";
describe("Setting Component", () => {
    it("renders correctly", () => {
        const { getByText } = render(<Setting />);
        expect(getByText("Setting")).toBeTruthy();
    });

    it("matches snapshot", () => {
        const tree = render(<Setting />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
