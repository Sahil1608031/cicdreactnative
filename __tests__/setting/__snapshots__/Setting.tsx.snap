// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setting Component matches snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Setting
  </Text>
</View>
`;
